src/index.js

```diff
@@ -61,9 +61,7 @@ const pixelRatio =
     ? window.devicePixelRatio
     : 1
 
-// Draws a rounded rectangle on a 2D context.
-const drawRoundedRect = (context, x, y, width, height, borderRadius) => {
-  if (borderRadius === 0) {
+const drawOpusBorder = (context, x, y, width, height, angleLeftRight = true) => {
```

Comment:
If we add Opus-specific features, will we still be able to pull updates from upstream?

Reply:
Yes, we can pull from upstream. We will have to fix any merge conflicts that occur. Probably what we should do is keep master clean and put our changes on a separate branch that we can install with package.json like so:
```
"react-avatar-editor": "git+https://github.com/OpusApps/react-avatar-editor.git#add-props-for-opus",
```
specifying the branch with
```
...git#add-props-for-opus
```

maybe call the branch `opus-prod`.

what do you think?

Reply:
Yep, that works for me.